src/models/GenericComponentDefinition.js

```diff
@@ -0,0 +1,23 @@
+import { ComponentDefinition } from 'leto-modelizer-plugin-core';
+
+/**
+ * Generic component definition.
+ */
+class GenericComponentDefinition extends ComponentDefinition {
+  /**
+   * Default constructor.
+   * @param {Object} [props] - Object that contains all properties to set.
+   * @param {boolean} [props.available=false] - Is these definitions is available to be used by end
```

Comment:
'**If** these definitions **are**...' or '**Are** these definitions available...'

Reply:
done

Reply:
`If this definition is...` for singular